package/MDAnalysis/coordinates/H5MD.py

```diff
@@ -0,0 +1,703 @@
+
+    >>> import MDAnalysis as mda
+    >>> from mpi4py import MPI
+    >>> u = mda.Universe("topology.tpr", "trajectory.h5md",
+    ...                   driver="mpio", comm=MPI.COMM_WORLD)
+
+.. Note::
+    h5py must be built with parallel features enabled on top of a parallel
+    HDF5 build, and HDF5 and mpi4py must be built with a working MPI
+    implementation. See instructions below.
+
+Building parallel h5py and HDF5
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+    1. `Build MPI from sources`_
+    2. `Build HDF5 from sources`_ with parallel settings enabled:
+
+        >>> ./configure --enable-parallel --enable-shared
+        >>> make
+        >>> make install
+
+    3. `Install mpi4py`_, making sure to point `mpicc` to where you've
+    installed your MPI implemenation:
+
+        >>> env MPICC=/path/to/mpicc pip install mpi4py
+
+    4. `Build h5py from sources`_, making sure to enable mpi and to point
+    to your parallel build of HDF5:
+
+        >>> export HDF5_PATH=path-to-parallel-hdf5
+        >>> python setup.py clean --all
+        >>> python setup.py configure -r --hdf5-version=X.Y.Z --mpi --hdf5=$HDF5_PATH
+        >>> export gcc=gcc
+        >>> CC=mpicc HDF5_DIR=$HDF5_PATH python setup.py build
+        >>> python setup.py install
+
+.. Warning::
+    You may have to play around with different combinations of
+    versions of h5py/HDF5 to get a working parallel build.
+
+.. _`H5MD`: https://nongnu.org/h5md/index.html
+.. _`HDF5`: https://www.hdfgroup.org/solutions/hdf5/
+.. _`H5PY`: http://docs.h5py.org/
+.. _`parallel h5py docs`: https://docs.h5py.org/en/stable/mpi.html
+.. _`mpi4py`: https://mpi4py.readthedocs.io/en/stable/index.html
+.. _`Build MPI from sources`: https://mpi4py.readthedocs.io/en/stable/appendix.html#building-mpi-from-sources
+.. _`Build HDF5 from sources`: https://support.hdfgroup.org/ftp/HDF5/current/src/unpacked/release_docs/INSTALL_parallel
+.. _`Install mpi4py`: https://mpi4py.readthedocs.io/en/stable/install.html#requirements
+.. _`Build h5py from sources`: https://docs.h5py.org/en/stable/mpi.html#building-against-parallel-hdf5
+.. _`H5MD notation`: https://nongnu.org/h5md/modules/units.html
+.. _`MDAnalysis notation`: https://userguide.mdanalysis.org/1.0.0/units.html
+
+
+Classes
+-------
+
+.. autoclass:: Timestep
+   :members:
+
+   .. attribute:: positions
+
+      coordinates of the atoms as a :class:`numpy.ndarray` of shape
+      `(n_atoms, 3)`
+
+   .. attribute:: velocities
+
+      velocities of the atoms as a :class:`numpy.ndarray` of shape
+      `(n_atoms, 3)`; only available if the trajectory contains velocities
+      or if the `velocities` = ``True`` keyword has been supplied.
+
+   .. attribute:: forces
+
+      forces of the atoms as a :class:`numpy.ndarray` of shape
+      `(n_atoms, 3)`; only available if the trajectory contains forces
+      or if the `forces` = ``True`` keyword has been supplied.
+
+
+.. autoclass:: H5MDReader
+   :members:
```

Comment:
Document the special `_reopen`
```suggestion
   :members:
   :private-members: _reopen
```
Check the docs and see if it appears.

Reply:
`:private-members: _reopen` wasn't working, it kept documenting ALL private methods even with `_reopen` passed.

This worked:
```
.. autoclass: H5MDReader
   :members:

      .. automethod:: H5MDReader._reopen
```